src/ngraph/util.hpp

```diff
@@ -38,6 +38,10 @@ namespace ngraph
     class NodeMap;
     class stopwatch;
 
+    // aliases for framework provided function pointers as defined in onnx runtime
+    using AllocateFunc = void* (*)(void*, size_t, size_t);
```

Comment:
Use std::function instead of C function pointers

Reply:
These should probably be defined in the backend.hpp file. They have meaning there.